tests/input_files/meshtying3D_sp_dual_simple.dat

```diff
@@ -86,11 +86,11 @@ SOLVER                          Superlu
 NAME                            Meshtying_Solver
 SOLVER                          Belos
 AZSOLVE                         GMRES
-AZPREC                          MueLu_contactSP
+AZPREC                          MueLu
 MUELU_XML_FILE                  meshtying3D_sp_simple.xml
 AZTOL                           1.0e-8
 AZCONV                          AZ_r0
-AZREUSE                         2
+AZREUSE                         1
```

Comment:
What's the reason behind changing this? The behavior of new and old should be the same.

Reply:
The reuse functionality is not working yet unfortunately. It is less straightforward to implement in the base class than in the MueLu_contactSP class, but I am working on it.

Reply:
@PhilipOesterlePekrun That's strange ... it should just work out of the box without any additional modifications.